friends/src/components/FriendList.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+
+const FriendList  = ({friend}, props) => {
+
+
+
+  const {id, name, age, email} = friend;
+
+
+  return (
+    <div className = 'friend-container'>
+      <div className = 'friend'>
+        <div className = 'friend_name'>
+          NAME: {name}
+        </div>
+
+        <div className = 'friend_age'>
+          AGE: {age}
+        </div>
+
+        <div className = 'friend_email'>
+          EMAIL: {email}
+        </div>
+     </div>
+
+     <button className = 'button_delete'  > Delete</button>
```

Comment:
add the delete functionality here onClick as a callback that takes  parameter of the friend ID